pkg/broker/update.go

```diff
@@ -120,6 +121,12 @@ func (b *broker) executeUpdatingStep(
 	}
 	// No next step-- we're done updating!
 	instanceCopy.Status = service.InstanceStateUpdated
+	// Merge the non-zero update parameters into the provision parameters
```

Comment:
i rebased off my other open PR and this came with it, so review this in the other one.